pkg/minikube/logs/logs.go

```diff
@@ -93,16 +94,16 @@ type logRunner interface {
 const lookBackwardsCount = 400
 
 // Follow follows logs from multiple files in tail(1) format
-func Follow(r cruntime.Manager, bs bootstrapper.Bootstrapper, cfg config.ClusterConfig, cr logRunner) error {
+func Follow(r cruntime.Manager, bs bootstrapper.Bootstrapper, cfg config.ClusterConfig, cr logRunner, logOutput io.Writer) error {
```

Comment:
is this PR also fixing the Follow ?  minikube logs -f has not been working

Reply:
I'm not sure how minikube logs -f is broken at the moment? As far as I can tell it's working properly on master.